app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -12,7 +12,8 @@ import org.oppia.app.settings.profile.ProfileListActivity
 import javax.inject.Inject
 
 const val SELECTED_CONTROLS_TITLE_KEY = "SELECTED_CONTROLS_TITLE_KEY"
```

Comment:
This need to be changed because this one is also getting used in `savedInstanceState`, check line `34` in code.

Reply:
done